src/index.md

```diff
@@ -58,6 +59,86 @@ const end = view(Inputs.date({label: "End", value: getDateXDaysAgo(1) }));
   }</div>
 </div>
 
+<div class="divider"></div>
+
+<h4>Spark Retrieval Result Codes</h4>
+<body>This section shows the Spark Retrieval Result Codes breakdown. You can adjust the date range. Records exist for the last 90 days.</body>
+
+```js
+const combine = (obj, target, keys) => {
+  obj[target] = keys.reduce((acc, key) => acc + (obj[key] || 0), 0)
+  for (const key of keys) {
+    delete obj[key]
+  }
+}
+const clone = obj => JSON.parse(JSON.stringify(obj))
+const tidy = clone(SparkRetrievalResultCodes).flatMap(({ day, rates }) => {
+  for (const [key, value] of Object.entries(rates)) {
+    rates[key] = Number(value)
+  }
+
+  combine(rates, 'HTTP_5xx', [
+    'LASSIE_502',
+    'HTTP_500',
+    'HTTP_502',
+    'ERROR_503',
+    'ERROR_522',
+    'ERROR_520',
+    'ERROR_500',
+    'BAD_GATEWAY',
+    'GATEWAY_TIMEOUT'
+  ])
+  combine(rates, 'HTTP_4xx', [
+    'ERROR_429',
+    'ERROR_404',
+    'ERROR_403',
+    'ERROR_408'
+  ])
+  combine(rates, 'IPNI_NOT_ADVERTISED', [
+    'IPNI_ERROR_404',
+    'IPNI_NO_VALID_ADVERTISEMENT',
+  ])
+  combine(rates, 'IPNI_4xx', [
+    'IPNI_ERROR_400',
+    'UNSUPPORTED_MULTIADDR_FORMAT'
```

Comment:
UNSUPPORTED_MULTIADDR_FORMAT is not an IPNI query error. It means the SP advertised retrievals with provider multiaddress we don't support.

Reply:
Got it! Fortunately this one was easy to put in the new `OTHER` category, which groups all currently insignificant errors